about.html

```diff
@@ -39,36 +39,39 @@
     </div>
   </section>
   <section id="about-info">
     <div class="wrapper">
-      <h2>Creative Commons Global Summit 2015 Logo</h2>
+      <h2>Post High-School Story</h2>
       <hr>
-      <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Consectetur, illo repudiandae architecto assumenda reiciendis natus consequuntur qui, rerum vero soluta quam harum placeat vitae eos repellat mollitia nostrum corporis molestiae.</p>
+      <p>At university. we found out that they gave scholarships for being part of their cultural groups. They had groups of dance, choir, painting, cinema and among those groups we found the ENSEMBLE DE CUERDAS that was directed by Iván Jumbo. That club had the premise of doing instrumental covers, with classical guitars only, of "Pasillo", Samba, Andean music, classical music and many other different genres of music. This extra-academic activity opened up our musical horizons and had a great influence on our current compositions.</p>
+      <p>At the same time, together with some friends, we created our first group named Doppler, with which we did several gigs and two original songs. We even took part in a music festival in Quito.</p>
+      <p>Now, we are developing new music combining this influences and learning every day. We are very excited to present our first album as Dasik, "Árbol Blanco".</p>
+      <p>Please contact us per email for any further questions about CC Global Summit 2015!</p>
+      <p><a href="mailto:dasikband@gmail.com" class="about-email">dasikband@gmail.com</a></p>
     </div>
   </section>
   <section id="past-events">
     <div class="wrapper">
-      <h2>See Past CC Global Summits</h2>
+      <h2>See Past Gigs</h2>
       <hr>
-      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ratione, culpa.</p>
+      <p>Check previous events and concerts.</p>
       <ul id="events">
         <li>
-          <img src="./images/event.jpg" alt="Event">
+          <img src="./images/LaAldeaGig.jpeg" alt="La Aldea">
           <div class="overlay">
-            <h3>2015</h3>
-            <p>Lorem ipsum dolor sit amet.</p>
+            <h3>2022</h3>
+            <p>"La Aldea" Pub Gig</p>
           </div>
         </li>
         <li>
-          <img src="./images/event.jpg" alt="Event">
+          <img src="./images/AfterGig2.jpeg" alt="After">
           <div class="overlay">
-            <h3>2013</h3>
-            <p>Lorem ipsum dolor sit amet.</p>
+            <h3>2021</h3>
+            <p>"After" Pub Gig</p>
           </div>
```

Comment:
- [ ] Great design and animations in this section 💯 It would be better to lay the red overlay with the text as default so as to follow the original design, then when a user hovers over the image, the red overlay and text then disappear.
- Your design:
![Annotation 2022-05-19 040902](https://user-images.githubusercontent.com/85113567/169197164-beedc65b-68fe-4c6c-8e40-3d0eb759e5fd.jpg)

- Required design:
![Annotation 2022-05-19 040840](https://user-images.githubusercontent.com/85113567/169197219-8eaf9dd5-991f-4432-a90f-fb708b2c3995.jpg)